test/python/utest_masarclient/test_timeStamp.py

```diff
@@ -0,0 +1,120 @@
+import unittest
+import time
+from types import *
+
+from masarclient.timeStamp import TimeStamp as TimeStamp
+
+timeStamp = TimeStamp()
+
+'''
+
+Unittests for masarService/python/masarclient/timeStamp.py
+
+'''
+
+
+class TestTimeStamp(unittest.TestCase):
+
+    '''
+    Sets up test variable for each test.
+    '''
+    def setUp(self):
+        self.test_time_stamp = TimeStamp()
+        #self.test_time_stamp.getCurrent()
+
+    '''
+    Ensures test_time_stamp is cleared after each test
+    '''
+    def tearDown(self):
+        self.test_time_stamp = 0
+
+    '''
+    Tests getter for time at the point of request
+    '''
+    def testGetCurrent(self):
+        self.test_time_stamp.getCurrent()
+        self.assertGreater(self.test_time_stamp.toSeconds(), 0.0, "Unexpected time returned")
+        test_stamp_two = TimeStamp()
+        test_stamp_two.getCurrent()
+        self.assertGreaterEqual(test_stamp_two, self.test_time_stamp, "Unexpected error in time continuity")
+
+    '''
+    This test can not confirm time frame, so it will just ensure the function doesn't crash
+    '''
+    def testGetSecondsPastEpoch(self):
+        self.test_time_stamp.getCurrent()
```

Comment:
Other than in ```testGetCurrent()``` it would be better to use ```TimeStamp.fromTime``` and work with instance with a known time.